grpc/src/test/java/com/linecorp/armeria/internal/server/grpc/GrpcDocServicePluginTest.java

```diff
@@ -375,9 +375,12 @@ void newStructInfo() throws Exception {
 
     @Test
     void httpEndpoint() {
-        final GrpcService grpcService =
-                GrpcService.builder().addService(mock(HttpJsonTranscodingTestServiceImplBase.class))
-                           .enableHttpJsonTranscoding(true).build();
+        final GrpcService grpcService = (GrpcService) GrpcService
+                .builder()
+                .addService(mock(HttpJsonTranscodingTestServiceImplBase.class))
+                .enableHttpJsonTranscoding(true)
+                .build()
+                .unwrap();
```

Comment:
This change could be reverted if https://github.com/line/armeria/pull/4041/files#r811707115 is applied.